src/packageurl/contrib/purl2url.py

```diff
@@ -314,6 +314,23 @@ def build_cocoapods_repo_url(purl):
     return name and f"https://cocoapods.org/pods/{name}"
 
 
+@repo_router.route("pkg:maven/.*")
+def build_maven_repo_url(purl):
+    """
+    Return a Maven repo URL from the `purl` string.
+    """
+    purl_data = PackageURL.from_string(purl)
+    namespace = purl_data.namespace
+    name = purl_data.name
+    version = purl_data.version
+
+    base_url = "https://repo1.maven.org/maven2"
+
+    if namespace and name and version:
+        maven_namespace = namespace.replace(".", "/")
+        return f"{base_url}/{maven_namespace}/{name}/{version}"
```

Comment:
Can you also handle the cases where you have qualifiers, such as for source or other maven "types"?